handler.go

```diff
@@ -124,6 +124,9 @@ func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request, next caddyht
 		if err != nil {
 			return err
 		}
+		// only close if there is no error; see PR #21
+		// at time of writing, Close() only flushes remaining bytes, but
```

Comment:
```suggestion
		// as of May 2023, Close() only flushes remaining bytes, but
```

I realized "at time of writing" is ambiguous as this is literally a Writer, lol